src/main/java/com/minecolonies/entity/ai/citizen/farmer/EntityAIWorkFarmer.java

```diff
@@ -568,4 +594,55 @@ public EntityCitizen getCitizen()
     {
         return worker;
     }
+
+    private boolean harvestCrop(final BlockPos position)
```

Comment:
This little thing might cause me some problems, I have to loop through the items to get rid of a seed, unless you don't care about that. Let me know @Minecolonies/maintainers.